### PR TITLE
Feature user logout

### PR DESCRIPTION
**Pull request by RainbowSeven**
User can log out of his/her session. Please merge.

[Review on Reviewable](https://reviewable.io/reviews/andela/troupon/32)

### CODE REVIEW

**Comment by RainbowSeven** [comment]
@andela-cnnadi All tests pass now. This branch can be merged.

File: troupon/.env.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DB_USER = "Administrator"
```
Comment (andela-cnnadi) [inline troupon/.env.py]: This file should be git ignored. Also plan to implement the module using YAML.

**Comment by RainbowSeven** [comment]
Fixes #30

**Comment by andela-cnnadi** [comment]
<sup>**[troupon/account/views.py, line 72 \[r4\]](https://reviewable.io:443/reviews/andela/troupon/32#-K-TB4hZ_IbJ8SgePcn9)** ([raw file](https://github.com/andela/troupon/blob/8f060438567159ba07f6fcc739aa99c76773e532/troupon/account/views.py#L72)):</sup>
Why are we not taking care of this exception?


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/andela/troupon/32)

**Comment by andela-cnnadi** [comment]
Reviewed 5 of 8 files at r3, 3 of 4 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/andela/troupon/32)

**Comment by andela-cnnadi** [comment]
Rebase and re-open pull request.

**Comment by andela-cnnadi** [comment]
You need to rectify conflicts on this branch to enable me merge.

**Comment by andela-osule** [comment]
Please merge this build. It adds the ability of a user to logout of the site.

**Pull request merged**